src/universal/services/ono/queries/pullrequests/index.js

```diff
@@ -112,18 +105,30 @@ export type RepositoryGraphType = {
   name: string
 }
 
+export const PullRequestSource = {
+  BRANCH: 'branch',
+  REV: 'rev',
```

Comment:
Is this the valid type? I think it should be BRANCH and REVISION. But you might need to test that manually...